astropy/modeling/blackbody.py

```diff
@@ -48,6 +48,26 @@
 >>> flux_nu  # doctest: +FLOAT_CMP
 <Quantity [  4.25135927e-123,  2.36894060e-005] erg / (cm2 Hz s sr)>
 
+Alternatively, the same results for ``flux_nu`` can be computed using ``BlackBody1D``
+from :class:`BlackBody1D` with blackbody representation as a model. The difference between
+this and the former approach is in one addition step as outlined as follows:
+
+>>> import numpy as np
```

Comment:
For better or worse, once imported above, this is already in the doctest namespace, thus repeating it is strictly unnecessary, especially since we are still in the same narrative flow.
```suggestion
```